run/helm/snap/charts/grafana/templates/job_cluster.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
```

Comment:
Because, unfortunately, using preloaded dashboard files does not support alerting. 

Preloaded dashboards may contain alerting rules in their json structure, but to fire those alerting rules you need to save the dashboard first (its ID must be saved in sqlite). And there is a bug in Grafana with handling saving of dashboards that are imported from files like that. 

What happens is when you try to save the dashboard in the UI, you create a copy of original dashboard. And you have two of them: copy and the original dashboard. Overwriting doesn't work. The saved copy doesn't contain alerting rules in its json anymore (it is just removed somehow while saving). And the original dashboard contains alerting rules, but still not triggered.

I got response to my issue from Grafana here: https://github.com/grafana/grafana/issues/7888

I know it's not elegant. It hurts me too. Alternatives know to me are:
* storing dashboards in Elasticsearch/InfluxDB
* using wizzy suggested by Grafana guy
* writing a script inside Grafana container

This solution seemed to me the most robust and the easiest to implement.